protobuf/flow/entities/execution_result.proto

```diff
@@ -10,7 +10,7 @@ message ExecutionResult {
   bytes block_id = 2;
   repeated Chunk chunks = 3;
   repeated ServiceEvent service_events = 4;
-  bytes execution_data_id = 5 [deprecated = true];
+  bytes execution_data_id = 5;
```

Comment:
```suggestion
  bytes execution_data_id = 5; // hash of execution data (internally partitioned chunk by chunk); requires additional consistency checks compared to including the execution data into their corresponding chunks directly
```

Reply:
I'd rather not add these to the public facing api documentation. they seem best targeted towards developers of the api implementation

Reply:
[66a5667](https://github.com/onflow/flow/pull/1574/commits/66a5667da50e891acdd259cd11ba68ceafc2e4eb)